openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx

```diff
@@ -95,17 +96,35 @@ const RichTextEditorPreviewer = ({
     <div
       className={classNames('rich-text-editor-container', className)}
       data-testid="viewer-container">
-      <div
-        className={classNames('markdown-parser', textVariant)}
-        data-testid="markdown-parser">
-        <Viewer
-          extendedAutolinks
-          customHTMLRenderer={customHTMLRenderer}
-          initialValue={viewerValue}
-          key={uniqueId()}
-          linkAttributes={{ target: '_blank' }}
-        />
-      </div>
+      <Popover
```

Comment:
If no tooltip then we should not be rendering the popover itself.

Reply:
@Sachin-chaurasiya  updated the changes as per comments!